src/store/modules/sitestatus/index.js

```diff
@@ -30,7 +30,7 @@ const state = {
 
   weather: {},
   enclosure: {},
-
+  owmReport: [],
```

Comment:
This variable name isn't super descriptive - maybe something like siteOwmReports would make more sense, as it's storing multiple owmReports per site.

Something to think about - would it make sense to have this be a JSON object, with each key being a site code? It would make the lookup simpler, so you can just do it by key.

e.g. 

```
{'sro1': {'report': ... ,
          'timestamp': ...},
 'aro': {'report': ... ,
         'timestamp': ...},
}
```